libwasmvm/src/handle_vm_panic.rs

```diff
@@ -31,4 +37,13 @@ mod tests {
         let err = catch_unwind(nice_try).unwrap_err();
         handle_vm_panic("nice_try", err);
     }
+
+    #[test]
+    fn can_handle_random_payloads() {
+        fn nice_try() {
+            std::panic::panic_any(());
```

Comment:
This shows `[unusable panic payload]`?

Reply:
Yep. Since the type is that's passed down is a `Box<()>` (with some dynamic dispatch indirection).

Just to verify that there's nothing panicking. Even though it never should panic of course.

Reply:
makes sense. Thanks for confirming